modules/os2web_acadre_esdh/os2web_acadre_esdh.module

```diff
@@ -6,6 +6,7 @@
  */
 
 define('OS2WEB_ACADRE_ESDH_ACADRE_MM_IMPORT_DIR', 'public://acadre');
+// @CODEREVIEW: What are these numbers? Should they be configurable?
```

Comment:
already configurable, that are the default values is no settings have been provided

Reply:
Thanks.